Sources/FoundationEssentials/CMakeLists.txt

```diff
@@ -55,21 +55,7 @@ if(SwiftFoundation_MACRO)
     target_compile_options(FoundationEssentials PRIVATE
```

Comment:
To keep the macro logic together, what do you think about moving this case up to `Sources/CMakeLists.txt` alongside the `ExternalProject` directives, and just creating an interface library that has this path? That way clients don't need to conditionalize on `SwiftFoundation_MACRO` and here we can just unconditionally link `FoundationBuildMacros`?

Reply:
Fixed